packages/date-picker/src/DatePicker/DateRangePicker.spec.tsx

```diff
@@ -27,6 +27,7 @@ describe("<DateRangePicker />", () => {
   })
 
   it("is able to select date range and shows in button", async () => {
+    jest.setTimeout(10000)
```

Comment:
I've added this here because running the full test suite locally sometime fails when the time to complete this test exceeds jests standard timeout
<img width="772" alt="Screen Shot 2022-04-21 at 1 44 21 pm" src="https://user-images.githubusercontent.com/36558508/164370084-36db7e63-1123-4a31-b041-c8e99ea440da.png">
s

Reply:
Interesting - Will look into a permeant fix later on.